app/helpers/action_links_helper.rb

```diff
@@ -50,6 +50,12 @@ def display_refund_link_for?(resource)
     can?(:refund, resource)
   end
 
+  def display_revert_link?
+    return false unless current_user
```

Comment:
I don't think we need to check for this – if no user is logged in, the user will be redirected anyway.

Reply:
good point! :D